kalite/contentload/management/commands/channels/base.py

```diff
@@ -215,41 +207,7 @@ def recurse_nodes_to_remove_childless_nodes(node):
             del node["children"][ci]
     recurse_nodes_to_remove_childless_nodes(topic_tree)
 
-    def recurse_nodes_to_add_position_data(node):
```

Comment:
It used to. It is no more now. And unnecessary, thanks to @66eli77's new knowledgemap.